server/middleware/populateCurrentUserRoles.ts

```diff
@@ -0,0 +1,12 @@
+import { RequestHandler } from 'express'
+import jwtDecode from 'jwt-decode'
+
+export default function populateCurrentUserRoles(): RequestHandler {
```

Comment:
This looks to be doing the same as as `server/middleware/authorisationMiddleware.ts` (already included in `app.ts`). Possible duplication?

Reply:
Removed this file and updated references to it.